src/layouts/styled.ts

```diff
@@ -0,0 +1,21 @@
+import { colors } from '../theme';
+import { createGlobalStyle } from 'styled-components';
+import styled from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
We can have only 1 Global Styles

Reply:
fix: Global Styles deleted, background set in the Container